src/v3/src/util/theme.ts

```diff
@@ -151,11 +152,21 @@ export const createThemeAndTokens = (
           }),
         },
       },
-      MuiInputBase: {
+      MuiFormHelperText: {
         styleOverrides: {
           root: {
-            width: '100%',
+            display: 'block',
           },
+        },
+      },
+      MuiInputBase: {
+        styleOverrides: {
+          root: ({ ownerState }) => ({
+            width: '100%',
+            ...(ownerState.name && isLtrField(ownerState.name) && {
+              direction: 'ltr',
```

Comment:
Makes all components that use `InputBase` under the hood (which happens to align nicely with all of the components that the SIW forces to be LTR) to be LTR at the theming layer since Odyssey does not support passing in direction props anywhere